### DISCUSSION TITLE
Hide footer row

### DISCUSSION THREAD

**Discussion by davegomez** [KevinVandy/material-react-table#1249] [Q&A] [answered]
Is there a way to hide or prevent rendering the footer row altogether if you do not use pagination like when using infinite scroll?

![footer](https://github.com/user-attachments/assets/c2872cf4-1ca9-47e3-9eb5-f04957c4d6a5)

**Comment by KevinVandy** [comment]
Do you mean the bottom toolbar? Cause yeah, there's a table option for that.

**Reply by davegomez** [reply]
Yes, that's the one I'm talking about.

But I haven't found the option.

**Reply by davegomez** [reply, marked as answer]
Ok I got it.

I wasn't thinking about it as a "toolbar".

https://www.material-react-table.com/docs/guides/toolbar-customization#hide-or-disable-toolbars
